modular_splurt/code/datums/traits/trait_actions.dm

```diff
@@ -1494,10 +1494,10 @@
 	button_icon_state = "blank"
 
 	// Glow color to use
-	var/glow_color = "#39ff14" // Neon green
+	var/glow_color
 
 	// Thickness of glow outline
-	var/glow_range = 1 //Less than radfiend
+	var/glow_range
```

Comment:
Since these values are used when first applying the glow, removing them will most likely break the quirk

Reply:
This is for tweak: Cosmetic Glow quirk now starts without the ugly green glow. (Works correctly, glow can still be applied)